go/libkb/upgrade_instructions.go

```diff
@@ -58,6 +58,16 @@ func darwinUpgradeInstructions() {
 	// TODO: non-brew update instructions
 }
 
+func windowsUpgradeInstructions() {
+	serverURI := ProductionServerURI
+	if DefaultRunMode == DevelRunMode {
+		serverURI = DevelServerURI
+	} else if DefaultRunMode == StagingRunMode {
+		serverURI = StagingServerURI
+	}
+
+	G.Log.Warning("To upgrade, download the latest Keybase installer from " + serverURI)
```

Comment:
I will do it hopefully tonight.  Will follow up.

On Sunday, November 29, 2015, Steve Sanders <notifications@github.com>
wrote:

> In go/libkb/upgrade_instructions.go
> <https://github.com/keybase/client/pull/1410#discussion_r46099964>:
>
> > @@ -58,6 +58,16 @@ func darwinUpgradeInstructions() {
> >  	// TODO: non-brew update instructions
> >  }
> >
> > +func windowsUpgradeInstructions() {
> > +	serverURI := ProductionServerURI
> > +	if DefaultRunMode == DevelRunMode {
> > +		serverURI = DevelServerURI
> > +	} else if DefaultRunMode == StagingRunMode {
> > +		serverURI = StagingServerURI
> > +	}
> > +
> > +	G.Log.Warning("To upgrade, download the latest Keybase installer from " + serverURI)
>
> ok, who will do the server side? or should I try doing it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1410/files#r46099964>.
>